packaging/installer-linux/installer-debian/src/main/resources/common/neo4j

```diff
@@ -46,12 +46,12 @@ SCRIPTNAME=/etc/init.d/$NAME-service
 
 do_start()
 {
-  start-stop-daemon --chuid ${NEO_USER} --start --quiet --oknodo --pidfile $PIDFILE --exec $DAEMON -- start
+  NEO4J_BIN=/usr/bin NEO4J_CONF=${NEO4J_CONF:-} start-stop-daemon --chuid ${NEO_USER} --start --quiet --oknodo --pidfile $PIDFILE --exec $DAEMON -- start
```

Comment:
I don't like this. @benbc do you have suggestions for cleaning this up, and the one below it?